source/install/config-mattermost.rst

```diff
@@ -0,0 +1,16 @@
+.. _config-mattermost:
```

Comment:
This file should be deleted. It was used when I had a separate sections for installing and for configuring.